### DISCUSSION TITLE
Scene is all black

### DISCUSSION THREAD

**Discussion by nullgogo** [mozilla/hubs#5925] [Art Questions] [answered]
**Description**
I convert an existing 3D model in FBX format to glb format through blender, and then load the converted glb format model into hubs as a scene through Import From Blender. When I use this scene, the scene is All black. 
And,  Remix in spoke is also  Error

**Screenshots**
 1.  3D in blender

![1](https://user-images.githubusercontent.com/52397681/215706504-4e5ec0fc-c991-4ac5-851e-78d6b3a87591.png)
 2.   hubs   Import From Blender
 
![2](https://user-images.githubusercontent.com/52397681/215706716-76bb9e04-bb0d-4829-a5bc-5d8652a03774.png)
![3](https://user-images.githubusercontent.com/52397681/215706776-8b95502f-8072-4710-ba76-58fdcd6186ca.png)
![4](https://user-images.githubusercontent.com/52397681/215706807-f7c1cbb3-1238-43c3-a1c0-209accaea60c.png)
 3. Scene is  all black
 
![5](https://user-images.githubusercontent.com/52397681/215706991-7a30cda0-6563-4eb0-aed6-c1c9c06e3ef3.png)
 4.    3D model  Remix in spoke  Error
 
![6](https://user-images.githubusercontent.com/52397681/215707165-f6bf6c6b-799e-4d90-8b7c-f395a40f26a0.png)
**Hardware** 
 - Device:  Desktop
 - OS: Windows
 - Browser: Firefox and chrome 

**Additional context**
Load the glb format model through the 3D viewer of windows, and the effect is as follows.

https://user-images.githubusercontent.com/52397681/215707456-0e55ca81-2f65-4436-8208-d37b89e7ea93.mp4

**Comment by LangDaoAI** [comment]
+1, The same scene also happened !

**Comment by matthewbcool** [comment, marked as answer]
@LangDaoAI @ChuangqHu  Double check you have at least one dynamic light or an environment map for lighting. We should expect the scene to be completely black if we have no lights or image-based lighting from the environment map. Use the 'import from blender' option as you did before. 👍

A few helpful tutorials to check out:
https://hubs.mozilla.com/labs/baked-lighting-for-better-looking-hubs-scenes/
https://hubs.mozilla.com/labs/creating-retro-worlds/

**Comment by LangDaoAI** [comment]
@matthewbcool thanks quick reply and suggestions! We will check and read the two tutorials carefully and will feedback you later.

**Reply by matthewbcool** [reply]
no problem! Hope that helps. If you are going to add a dynamic light, keep in mind lights have a pretty big impact on performance and you will want to remember to use the Hubs component and export 'punctual lights' in blender. 
<img width="226" alt="lights" src="https://user-images.githubusercontent.com/4493657/215983373-03c7ce32-611e-482e-886c-73d485246902.png">

Adding an Env map gives you quick image based-lighting:
<img width="286" alt="image" src="https://user-images.githubusercontent.com/4493657/215985729-85ba6574-0572-4aaa-9ef4-94ab324fb2e2.png">

**Comment by LangDaoAI** [comment]
@matthewbcool you are welcome! and thanks your detailed suggestions again!

**Comment by nullgogo** [comment]
Still  Black

**Here is my operation flow**

1.   Add  sun
![add sun](https://user-images.githubusercontent.com/52397681/216903341-d4887e89-b120-486a-991b-b7d6d04a84f0.png)

2.  Add  new UVMap
![add  a new  'UVMap'](https://user-images.githubusercontent.com/52397681/216903463-2e895b9f-3440-4a5a-84cb-b70c627a06c9.png)

3. Shader Editor
![editor1](https://user-images.githubusercontent.com/52397681/216903531-9ff2a930-b308-4eb3-ae46-4202ef6383fe.png)
![editor2](https://user-images.githubusercontent.com/52397681/216903539-6c8751b6-847b-4f30-b4d8-687520e4c9b8.png)

4. export
![export](https://user-images.githubusercontent.com/52397681/216903606-25f680cc-6006-4bfa-b308-7f855671dcf9.png)

**Reply by nullgogo** [reply]
@matthewbcool

**Comment by nullgogo** [comment]
I've got it ，thanks

**Reply by matthewbcool** [reply]
aweseome! just in case others stumble on this thread, the answer was likely you need to add a light component with the Hubs exporter to the sun. Was that it?

**Reply by nullgogo** [reply]
Yes, that's exactly it
